pkg/run/run.go

```diff
@@ -44,14 +44,15 @@ func (b *DockerRunner) Run(config *api.Config) error {
 
 	opts := docker.RunContainerOptions{
 		Image:        config.Tag,
+		Entrypoint:   []string{"/bin/env"},
```

Comment:
If we always (?) use `/bin/env`, then maybe make it default? Or it's impossible/undesirable?

(BTW I've never heard about `/bin/env` before and in many examples  `/usr/bin/env` is used.)